### PR TITLE
{vis} [intel/2016a]  ImageMagick 7.0.1 

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.19-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+name = 'Ghostscript'
+version = '9.19'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+# lowopt (-O1) is used to avoid internal compiler error that is hit when -O2 is used
```
Comment (boegel) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.19-intel-2016a.eb]: @wpoely86 check if this is still needed?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in this PR)
node2648.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/87af1ae19ea4551ca6215cd363efcf66 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node2648.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d486b2bfd288ce638367330f749537a0 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2648.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a88b41c6c3b673cb573808e7021e42e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f5976b147402ff6df8ac8cc7520723a9 for a full test report.

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.19-intel-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+version = '9.19'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ["https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs919/"]
```
Comment (boegel) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.19-intel-2016a.eb]: @wpoely86 don't hardcode the version in here please, use `%(version_major)s%(version_minor)s`

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**